pandas/core/indexes/datetimes.py

```diff
@@ -2245,17 +2245,47 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     --------
     pandas.period_range : Return a fixed frequency PeriodIndex.
     pandas.interval_range : Return a fixed frequency IntervalIndex.
+    Numpy daterange: `numpy daterange<https://docs.scipy.org/doc/numpy/reference/arrays.datetime.html>`__.
```

Comment:
See Also is for specific methods, classes, or modules. I would move this link to the extended summary, along with the note about frequency strings.